### PR TITLE
Accept Htmlable for table column label

### PR DESCRIPTION
**Pull request by Tabitha-art**
added to the property and setter as well

### CODE REVIEW

File: packages/tables/src/Columns/Concerns/HasLabel.php [hunk 1]
```diff
@@ -4,14 +4,15 @@
 
 use Closure;
 use Illuminate\Support\Str;
+use Illuminate\Contracts\Support\Htmlable;
```
Comment (zepfietje) [inline packages/tables/src/Columns/Concerns/HasLabel.php]:
```suggestion
use Illuminate\Contracts\Support\Htmlable;
use Illuminate\Support\Str;
```

**Comment by zepfietje** [comment]
Doesn't this break the `shouldTranslateLabel` functionality, @Tabitha-art?

**Comment by danharrin** [comment]
@zepfietje I don't think it breaks that, they just don't work together which isn't a bad thing. I don't even like `translateLabel()` lol.

**Comment by zepfietje** [comment]
Me neither. 😂

**Pull request merged**